helpers/slots.js

```diff
@@ -38,15 +38,12 @@ function getEpochTime(time) {
 		time = Date.now();
 	}
 
-	var d = constants.EPOCH_TIME;
-	var t = d.getTime();
-
-	return Math.floor((time - t) / 1000);
+	return Math.floor((time - EPOCH_TIME.getTime()) / 1000);
 }
 
 module.exports = {
 	interval: 10,
-	delegates: constants.ACTIVE_DELEGATES,
+	delegates: ACTIVE_DELEGATES,
```

Comment:
We shouldn't export `ACTIVE_DELEGATES` as `delegates` here and use `ACTIVE_DELEGATES` in corresponding modules that uses `slots.delegates`, can we fix this?

Reply:
this requires a big refactor. Let's do it in a separate issue. See: https://github.com/LiskHQ/lisk/issues/2413

Reply:
@4miners see my comment above.

Reply:
Ahh, ok, sorry. 🤗